app/views/topical_events/show.html.erb

```diff
@@ -123,7 +123,7 @@
   </div>
 
   <div class="govuk-grid-column-two-thirds">
-    <% if @topical_event.organisations %>
+    <% if @topical_event.emphasised_organisations && @topical_event.emphasised_organisations.any? %>
```

Comment:
I wonder if it might be better to just add this section in the final commit, rather than introducing it earlier?  I got slightly confused looking at the diff as to what changed each time.

Reply:
Resolved this by just squashing the last two commits together, they weren't really distinct from each other